build/gulp/plugins/gulp-react-docgen.ts

```diff
@@ -19,11 +19,18 @@ export default () =>
       return
     }
 
-    try {
-      const contents = getComponentInfo(file.path)
+    const infoFilename = file.basename.replace(/tsx$/, 'info.json')
+    const nextChecksum = getBufferChecksum(file.contents)
 
+    if (getInfoChecksum(infoFilename) === nextChecksum) {
```

Comment:
I don't understand what you want to do there. `getInfoChecksum()` will be always called for `*.info.json`, how are you planning to reduce reads?

Reply:
the reason I've asked is because seen `try/catch` block in `checksumUtils.ts` - have clarified which cases we are capturing there. If this is only about `non-existing file`, then we are fine :)